### ISSUE TITLE
add Meetup icon to footer

### ISSUE THREAD

**Issue by M0nica** [M0nica/React-Ladies#28]
Add the Meetup icon to the footer and link to the Meetup group's page: https://www.meetup.com/react-ladies

**Comment by antropova** [comment]
@M0nica where do you download your svg's from?

**Comment by M0nica** [comment]
@antropova I don't think that I've added Font Awesome to this project yet but I usually use a source like Font Awesome.  https://fontawesome.com/icons/meetup. 

Meetup also has .ai vector files of their logo available here: https://www.meetup.com/media/

**Comment by kevinsunny1996** [comment]
@M0nica  can I take up this issue?...I would like to give it a try.

**Comment by ChenyunZhang** [comment]
Hi, Monica, is this issue taken? if not, I would like to take this, thank you.

**Comment by M0nica** [comment]
> Hi, Monica, is this issue taken? if not, I would like to take this, thank you.

Okay, great this is still available.
